modules/apps/foundation/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/liferay/app/App.es.js

```diff
@@ -110,6 +113,11 @@ class LiferayApp extends App {
 			}
 		);
 
+		if (!this.pendingNavigate) {
```

Comment:
For this check, I think it would be better to use the `isNavigationPending` flag in senna, but the version that is in portal does not have that flag yet.  If you think it's acceptable to upgrade senna in portal at this time, maybe we can change this as well.